src/components/Navbar/Drawer/index.tsx

```diff
@@ -83,6 +83,7 @@ const Drawer: React.FC<Props> = ({
   hideCloseButton = false,
   closeOnNavigation = true,
 }) => {
+  const { isVisible } = useSelector(selectNavbar, shallowEqual);
```

Comment:
`const isNavbarVisible = useSelector(selectNavbar, shallowEqual).isVisible;`

If you use isVisible variable it might be confused for the drawer being visible. Feel free to merge after :)